lib/ui/pointer.dart

```diff
@@ -178,11 +179,15 @@ class PointerData {
     this.rotation = 0.0,
   });
 
-  /// Unique identifier that ties the [PointerEvent] to embedder event created it.
+  /// The ID of the [FlutterView] this [PointerEvent] originated from.
+  final int viewId;
+
+  /// Unique identifier that ties the [PointerEvent] to embedder event created
```

Comment:
Nit:
```suggestion
  /// Unique identifier that ties the [PointerEvent] to the embedder
  /// event that created it.
```

(My formatting may be off)